libs/domain/search/box/box.component.spec.ts

```diff
@@ -96,9 +96,9 @@ describe('SearchBoxComponent', () => {
     linkService = testInjector.inject<MockLinkService>(
       LinkService
     ) as MockLinkService;
-    routerService = testInjector.inject(
+    routerService = testInjector.inject<MockRouterService>(
```

Comment:
Should be:
```suggestion
    routerService = testInjector.inject<MockRouterService>(RouterService);
```
Without type casting. And check the other injections behind this one